client/my-sites/plans/current-plan/index.jsx

```diff
@@ -164,12 +164,17 @@ class CurrentPlan extends Component {
 		} = this.props;
 
 		const currentPlanSlug = selectedSite.plan.product_slug,
-			isLoading = this.isLoading();
-
-		const planConstObj = getPlan( currentPlanSlug ),
-			planFeaturesHeader = translate( '%(planName)s plan features', {
-				args: { planName: planConstObj.getTitle() },
-			} );
+			isLoading = this.isLoading(),
+			planTitle = getPlan( currentPlanSlug ).getTitle();
+
+		const planFeaturesHeader =
+			typeof planTitle === 'string'
+				? translate( '%(planName)s plan features', {
+						args: { planName: planTitle },
+				  } )
+				: translate( '{{planName/}} plan features', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( '%(planName)s plan features' )` *ES Score: 11*

Reply:
You could even probably drop the condition altogether with `translate( '{{planName/}} plan features', { components: { planName: <>planTitle</> }} );`

Reply:
Yes, good idea, that look much better! :)